TramsDataApi.Test/Factories/AcademyConversionProjectFactoryTests.cs

```diff
@@ -104,57 +104,69 @@ private AcademyConversionProject CreateAcademyConversionProject()
-            expected.PreviousHeadTeacherBoardDate = updateRequest.PreviousHeadTeacherBoardDate;
-            expected.SchoolAndTrustInformationSectionComplete = updateRequest.SchoolAndTrustInformationSectionComplete;
-            expected.DistanceFromSchoolToTrustHeadquarters = updateRequest.DistanceFromSchoolToTrustHeadquarters;
-            expected.DistanceFromSchoolToTrustHeadquartersAdditionalInformation = updateRequest.DistanceFromSchoolToTrustHeadquartersAdditionalInformation;
-            expected.MemberOfParliamentParty = updateRequest.MemberOfParliamentParty;
-            expected.MemberOfParliamentName = updateRequest.MemberOfParliamentName;
-            expected.GeneralInformationSectionComplete = updateRequest.GeneralInformationSectionComplete;
-            expected.RisksAndIssuesSectionComplete = updateRequest.RisksAndIssuesSectionComplete;
-            expected.SchoolPerformanceAdditionalInformation = updateRequest.SchoolPerformanceAdditionalInformation;
-            expected.CapitalCarryForwardAtEndMarchCurrentYear = updateRequest.CapitalCarryForwardAtEndMarchCurrentYear;
-            expected.CapitalCarryForwardAtEndMarchNextYear = updateRequest.CapitalCarryForwardAtEndMarchNextYear;
-            expected.SchoolBudgetInformationAdditionalInformation = updateRequest.SchoolBudgetInformationAdditionalInformation;
-            expected.SchoolBudgetInformationSectionComplete = updateRequest.SchoolBudgetInformationSectionComplete;
-            expected.SchoolPupilForecastsAdditionalInformation = updateRequest.SchoolPupilForecastsAdditionalInformation;
-            expected.YearOneProjectedCapacity = updateRequest.YearOneProjectedCapacity;
-            expected.YearOneProjectedPupilNumbers = updateRequest.YearOneProjectedPupilNumbers;
-            expected.YearTwoProjectedCapacity = updateRequest.YearTwoProjectedCapacity;
-            expected.YearTwoProjectedPupilNumbers = updateRequest.YearTwoProjectedPupilNumbers;
-            expected.YearThreeProjectedCapacity = updateRequest.YearThreeProjectedCapacity;
-            expected.YearThreeProjectedPupilNumbers = updateRequest.YearThreeProjectedPupilNumbers;
-            expected.KeyStage2PerformanceAdditionalInformation = updateRequest.KeyStage2PerformanceAdditionalInformation;
-            expected.KeyStage4PerformanceAdditionalInformation = updateRequest.KeyStage4PerformanceAdditionalInformation;
-            expected.KeyStage5PerformanceAdditionalInformation = updateRequest.KeyStage5PerformanceAdditionalInformation;
-            expected.ConversionSupportGrantAmount = updateRequest.ConversionSupportGrantAmount ?? default;
-            expected.ConversionSupportGrantChangeReason = updateRequest.ConversionSupportGrantChangeReason;
-            expected.ProjectStatus = updateRequest.ProjectStatus;
-
-            return expected;
+           var expected =
+              JsonConvert.DeserializeObject<AcademyConversionProject>(JsonConvert.SerializeObject(original));
```

Comment:
It's a bit tricky to tell what's changed here.
Was it just the order of the assertions?

Reply:
Mostly formatting (wish it would ignore whitespace) Lines 123-127 are new additions.